README.md

```diff
@@ -614,8 +616,196 @@ let sameFont = UIFont(font: FontFamily.Helvetica.Bold, size: 16.0) // iOS
 let sameFont = NSFont(font: FontFamily.Helvetica.Bold, size: 16.0) // OS X
 ```
 
----
+## Models
+
+```sh
+swiftgen json /path/to/model/file -t model
+swiftgen templates cat model-base > /path/to/output/file
+```
+
+This takes a JSON spec and converts it into a model file which supports parsing from a dictionary, converting back into a dictionary, equality, mapping, and three different initialization options.
+
+The input is a JSON file which creates a spec for the object. The schema is a custom schema but is based off a few standards (AVRO, Pegasus, JSON Schema). The spec takes a type name and a list of fields. Each field specifies it's type, name, whether it's optional and a doc. It can also specify `array` and `element` for an array of elements. The type and element field can be anything. The model generator simply converts it to an uppercase string for the type.
+
+The model template also requires some static code which makes the generated code smaller. This static code is outputed by `swiftgen templates cat model-base > /path/to/output/file`. You should generate this code as well as the model code for each of your JSON specs.
```

Comment:
See above comment, this could simply become "Add the line `pod 'MyMiniParsingLibrary'` to your Podfile or library of choice".